src/main/webapp/new_post.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset="UTF-8">
+    <title>New Post</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    </head>
+
+
+    <body>
+        <h1>Create new Post</h1>
+        <hr>
+        <form>
+            <input type="text" id="title" placeholder="Title"><br>
```

Comment:
It would also be very good if all form input tags have a label like how you did for the one below.